docker-compose.yml

```diff
@@ -48,10 +48,9 @@ services:
       - BAMBORA_PROFILE_SERVICE_VERSION=${BAMBORA_PROFILE_SERVICE_VERSION}
       - BAMBORA_URL_EXPIRY=10
 
-      - KEYCLOAK_AUTH_SERVER_URL=${KEYCLOAK_AUTH_SERVER_URL:-http://keycloak:8080/auth}
-      - KEYCLOAK_REALM=${KEYCLOAK_REALM:-Efiling-Hub}
-      - KEYCLOAK_CREDENTIALS_SECRET=${KEYCLOAK_CREDENTIALS_SECRET}
-      - KEYCLOAK_SSL_REQUIRED=${KEYCLOAK_SSL_REQUIRED:-none}
+      - KEYCLOAK_AUTH_SERVER_URL=${KEYCLOAK_AUTH_SERVER_URL:-http://localhost:8081/auth/realms/Efiling-Hub}
+      - KEYCLOAK_JWK_SERVER_URL=${KEYCLOAK_JWK_SERVER_URL:-http://keycloak:8080/auth/realms/Efiling-Hub}
```

Comment:
This is a work around. Because our automation suite is run in docker we have to use a different value for the server and issuer. In actual deployment these values will be the same.